engine/robot.py

```diff
@@ -126,10 +118,9 @@ def __init__(self, x_pos, y_pos, heading, epsilon, max_v, radius, is_sim=True, p
             output_limits=(None, None)
         )
 
-        cwd = os.getcwd()
-        cd = cwd + "/csv"
+        # TODO: wrap in try/except (error when calling execute_setup_test.py)
```

Comment:
Let's keep these todo comments in a separate pr next time

Reply:
^! we dont want to forget them so make sure these TODOs are mentioned in some sort of future steps documentation / mentioned in person to members